cmd/blockio/main.go

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright 2021 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// This application demonstrates using the blockio API.
+
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/intel/goresctrl/pkg/blockio"
+)
+
+var examples string = `Examples:
+    # Inspect OCI blockio structure
+    $ blockio -config sample.cfg -class slowread | jq
+
+    # Apply read throttling to a cgroup
+    $ blockio -config sample.cfg -class slowread -cgroup user.slice/mygroup
+    $ cat /sys/fs/cgroup/blkio/user.slice/mygroup/blkio.throttle.read_bps_device
+
+    # Remove throttling from a cgroup
+    $ blockio -config sample.cfg -class nolimit -cgroup user.slice/mygroup
+`
+
+func usage() {
+	flag.CommandLine.SetOutput(os.Stdout)
+	fmt.Fprintln(flag.CommandLine.Output(), "blockio - demo application for goresctrl/pkg/blockio API")
+	fmt.Fprintln(flag.CommandLine.Output(), "Usage: blockio [options]")
```

Comment:
nit: I think `[options]` is a bit misleading because `-config` and `-class` seem to be mandatory.

Reply:
Thanks! Usage rewritten.